lib/send-mail.ts

```diff
@@ -54,7 +54,7 @@ export async function parseMBox(mbox: string, gentle?: boolean): Promise<IParsed
 
     for (const entry of parsed.headerLines) {
         // try to parse header line and consume a leading line break after the colon in folded headers
-        const valueSet = entry.line.match(/(.*?):(?:\r?\n)? *([^]*)$/);
+        const valueSet = entry.line.match(/(.*?):(?:\r\n)? *([^]*)$/);
```

Comment:
I do not quite understand why we cannot be lenient here, but I guess it does not matter much in practice 😉